src/Microsoft.AspNet.Mvc.ViewFeatures/ViewFeatures/FormContext.cs

```diff
@@ -29,6 +29,14 @@ public class FormContext
             }
         }
 
+        public bool HasFormData
```

Comment:
It's less verbose. No reason not to adopt features when they are the better alternative.